setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import setuptools
+
+
+with open("README.md", 'r') as f:
+    long_description = f.read()
+
+setuptools.setup(
+        name="garnish",
+        version="0.2-alpha",
```

Comment:
```suggestion
        version="0.3-alpha",
```

Reply:
I can't approve the change for some reason...